source/draw/gpu/GpuCanvasYuv420Semiplanar.ooc

```diff
@@ -76,9 +76,10 @@ GpuCanvasYuv420Semiplanar: class extends GpuCanvas {
 		uvLines free()
 	}
 	drawPoints: override func (pointList: VectorList<FloatPoint2D>) { this _target y canvas drawPoints(pointList) }
-	fill: override func {
-		this _target y canvas fill()
-		this _target uv canvas fill()
+	fill: override func ~color (color: ColorRgba) {
+		yuv := color toYuv()
+		this _target y canvas fill(ColorRgba new(yuv y, 0, 0, 255))
+		this _target uv canvas fill(ColorRgba new(yuv u, yuv v, 0, 255))
```

Comment:
Shouldn't this conversion be applied in the canvas for Y and UV?
```ooc
fill: func (color: ColorRgba) {
    yuv := color toYuv()
    this _backend setClearColor(yuv y)
}
```
something like that.